template.rb

```diff
@@ -3,54 +3,34 @@
 #
 
 #
-# Authorization / Access Control
+# Authorization
 #
-if yes?('Would you like to install Devise?')
-  # Authorization
-  gem 'devise'
-  generate 'devise:install'
-  model_name = ask('What would you like the user model to be called? [User]')
-  model_name = 'User' if model_name.blank?
-  generate 'devise', model_name
-  generate 'devise:views' if yes?('Install Devise view files?')
-
-  # Access Control
-  acl = ask('Which ACL do you want? [1] = CanCan [2] = CanCanCan [3] = Pundit')
-  case acl
-  when '2'
-    gem 'cancancan', '~> 1.10'
-    generate 'cancan:ability'
-  when '3'
-    gem 'pundit'
-    generate 'pundit:install'
-  else
-    gem 'cancan'
-    generate 'cancan:ability'
-  end
-end
+gem 'devise'
+generate 'devise:install'
+model_name = ask('What would you like the user model to be called? [User]')
+model_name = 'User' if model_name.blank?
+generate 'devise', model_name
+generate 'devise:views' if yes?('Install Devise view files?')
+# Access Control
+gem 'pundit'
+generate 'pundit:install'
 
 # Setup Static Home Page
 generate 'controller', 'static home'
 route "root to: 'static#home'"
 gsub_file 'config/routes.rb', /^  get 'static\/home'\n$/, ''
 
+# Twitter Bootstrap
+gem 'bootstrap', '~> 4.0.0.alpha6'
 # Pagination
 gem 'kaminari'
-# HTTP server currently preferred by Heroku
-gem 'puma'
-gem 'jquery-rails'
-gem 'jquery-turbolinks'
```

Comment:
Pulling, we don't know if its necessary now. Ember + Rails... I've read that Turbolinks is problematic and might not be necessary.